_observing-your-data/query-insights/grouping-top-n-queries.md

```diff
@@ -45,6 +45,96 @@ bool
 
 When queries share the same query structure, they are grouped together, ensuring that all similar queries belong to the same group.
 
+## Configuring query structure
+
+The preceding example query shows a simplified query structure. By default, the query structure also includes field names and field data types. 
+
+For example, consider an index `index1` with the following field mapping:
+
+```json
+"mappings": {
+  "properties": {
+    "field1": {
+      "type": "keyword"
+    },
+    "field2": {
+      "type": "text"
+    },
+    "field3": {
+      "type": "text"
+    },
+    "field4": {
+      "type": "long"
+    }
+  }
+}
+```
+
+The following query on the preceding index:
```

Comment:
This needs a little more context/clarity of phrasing to indicate that it's connected to the next sentence and not the preceding one.

Reply:
@natebower Reworded - please check.